concent_api/concent_api/system_check.py

```diff
@@ -129,3 +137,13 @@ def storage_cluster_certificate_path_check(app_configs = None, **kwargs):  # pyl
         elif not os.path.isfile(certificate_path):
             errors.append(create_error_15_ssl_cert_path_is_not_a_file(certificate_path))
     return errors
+
+
+@register()
+def check_atomic_requests(app_configs = None, **kwargs):  # pylint: disable=unused-argument
```

Comment:
could you please add some unit tests to test the logic of the function?
some examples for other `system_check`: https://github.com/golemfactory/concent/blob/master/concent_api/concent_api/tests/test_storage_cluster_certificate_path_check.py